### ISSUE TITLE
As a network starter, I can start a business network in a consortium

### ISSUE THREAD

**Issue by sstone1** [hyperledger/composer#1293]
In Hyperledger Fabric v0.6, the deploy processing was easy. The network starter sent a single deploy transaction to the Blockchain which included the Composer runtime (chaincode/smart contract) and the business network definition. This transaction installed the Composer runtime onto all of the peer nodes and started it. 

In Hyperledger Fabric v1.0, this process is now a lot more involved. There are now multiple actions - some that are performed within an organisation, some that are performed across a consortium (a set of organisations). The network start process is now:

1. Each organisation in the business network must **install** the Composer runtime on all of their endorsing peer nodes. This simply installs the Composer runtime into the file system of those endorsing peer nodes, nothing starts running at this point. The user performing this action must be a peer administrator.
2. Someone from the consortium must **instantiate** the Composer runtime across all of the endorsing peer nodes in the network. This is a single transaction and is subject to an instantiation policy which is configured by the administrators who set up the network in the first place. The transaction starts the installed Composer runtime and calls the `Init` function. The instantiation policy defines which users are able to perform this action.

At a later point in time, the consortium may decide to move to a later version of the Composer runtime to pick up new functionality and fixes. This is a similar process to network start:

1. Each organisation in the business network must **install** the **new version** of the Composer runtime on all of their endorsing peer nodes. This simply installs the new version of the Composer runtime into the file system of those endorsing peer nodes, nothing starts running at this point. The existing instantiated version remains running. Again, the user performing this action must be a peer administrator.
2. Someone from the consortium must **upgrade** the Composer runtime across all of the endorsing peer nodes in the network. This is a single transaction and is subject to an instantiation policy which is configured by the administrators who set up the network in the first place. The transaction starts the the new version of the installed Composer runtime and calls the `Init` function. Importantly the world state and ledger remain in the same state. Again, the instantiation policy defines which users are able to perform this action.

Note that one _interesting_ design decision from the Hyperledger Fabric team is the lack of ability to install the Composer runtime once and instantiate it with multiple names. This means that we have no choice but to install the Composer runtime for each business network, and give it the name of the business network. We have a JIRA open for this: https://jira.hyperledger.org/browse/FAB-3016. The dream is to be able to install the Composer runtime once, maybe even preinstall it, and then just instantiate it for each business network.

This user story is specifically focusing on separating the second step, instantiate, from the deploy processing so that someone from the consortium can run it across all of the endorsing peer nodes in this network.

Let's get the controversial bit out of the way first; `instantiate` sucks as a term. It's long, difficult to spell, and it doesn't really map that well to what is happening. 

I believe that `start` is a much better term - it's short, snappy, and is easy to explain to both business and technical users of Composer.

Moving swiftly on, I propose the following changes:

1. Add `start` to the connector API, `Connection.start(businessNetworkDefinition)`.
2. Expose `start` through the administrative API, `AdminConnection.start(businessNetworkDefinition)`.
3. Expose `start` through the CLI, `composer network start`.

I do not propose removing `deploy` as part of this work - we should simply update `deploy` to call the new `start` API. Deprecation and removal of `deploy` can come at a later stage once the changes have "bedded in".

**Comment by davidkel** [comment]
Same as install, we will need startOptions as part of the API.

**Comment by davidkel** [comment]
```
cli.js runtime start [options]

Options:
  --help                       Show help  [boolean]
  -v, --version                Show version number  [boolean]
  --archiveFile, -a            The business network archive file name  [string] [required]
  --connectionProfileName, -p  The connection profile name  [string]
  --loglevel, -l               The initial loglevel to set (INFO|WARNING|ERROR|DEBUG)  [string]
  --option, -o                 Options that are specific specific to connection. Multiple options are specified by repeating this option  [string]
  --optionsFile, -O            A file containing options that are specific to connection  [string]
  --enrollId, -i               The ID of the permitted start user  [string] [required]
  --enrollSecret, -s           The Secret of the permitted start user, if required  [string]
```
Same comment about enrollId and enrollSecret as with install, may not be the best choice of parameter names and descriptions. Please feel free to provide replacements

**Comment by davidkel** [comment]
Closing as now released in 0.10.0
